<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxSensors">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>EEG Site Contact</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="5">
       <widget class="QCheckBox" name="checkBoxSensor_6">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="checkBoxSensor_7">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="checkBoxSensor_14">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBoxSensor_10">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxSensor_1">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBoxSensor_11">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxSensor_9">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBoxSensor_4">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="checkBoxSensor_13">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBoxSensor_12">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBoxSensor_2">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkBoxSensor_5">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="checkBoxSensor_15">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxSensor_3">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QCheckBox" name="checkBoxSensor_8">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QCheckBox" name="checkBoxSensor_16">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxSensor_17">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBoxSensor_18">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBoxSensor_19">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="checkBoxSensor_20">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="checkBoxSensor_21">
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonSensorsOn">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>All On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSensorsOff">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>All Off</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSensors">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>116</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>254</width>
       <height>49</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">&lt;link href=&quot;https://fonts.googleapis.com/icon?family=Material+Icons&quot; rel=&quot;stylesheet&quot;&gt;&lt;i class=&quot;material-icons&quot;&gt;pause&lt;/i&gt;</string>
        </property>
        <property name="text">
         <string>▮▮</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>▷</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>▢</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="sessionWidget" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>201</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="sessionLayout">
      <item>
       <widget class="QLabel" name="description">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timer">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>progressBar</zorder>
     <zorder>timer</zorder>
     <zorder>description</zorder>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>39</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">&lt;link href=&quot;https://fonts.googleapis.com/icon?family=Material+Icons&quot; rel=&quot;stylesheet&quot;&gt;&lt;i class=&quot;material-icons&quot;&gt;pause&lt;/i&gt;</string>
     </property>
     <property name="text">
      <string>☰</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="screen">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
    </widget>
    <widget class="QWidget" name="blueLight" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gray;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>82</width>
       <height>118</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="upButton">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>▲</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downButton">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>▼</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="menuWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>NEW SESSION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SESSION LOG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TIME AND DATE</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="greenLight" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gray;</string>
     </property>
    </widget>
    <widget class="QWidget" name="redLight" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gray;</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="battery">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>350</y>
       <width>41</width>
       <height>91</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk {
	border: 2px solid black;
	background-color: lightgreen;
}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>51</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;</string>
     </property>
     <property name="text">
      <string>O
I</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateTimeEdit {
	border: none;
	background-color: transparent;
}

QDateTimeEdit::up-button {
	width: 60px;
	background-color: transparent;
}

QDateTimeEdit::down-button {
	width: 60px;
	background-color: transparent;
}</string>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd h:mm AP</string>
     </property>
    </widget>
    <widget class="QListWidget" name="sessionLog">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>menuButton</zorder>
    <zorder>screen</zorder>
    <zorder>blueLight</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>menuWidget</zorder>
    <zorder>greenLight</zorder>
    <zorder>redLight</zorder>
    <zorder>sessionWidget</zorder>
    <zorder>battery</zorder>
    <zorder>powerButton</zorder>
    <zorder>dateTimeEdit</zorder>
    <zorder>sessionLog</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>External factors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="replaceBatteryButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace Battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testGamma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Gamma Wave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testTheta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Theta Wave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testBeta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Beta Wave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testDelta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Delta Wave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testAlpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Alpha Wave</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
